index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+    <main>
+        <section class="container">
+                <header class="nav_bar">
+                        <a class="myLogo" href="#"><img src="image/My_Logo.svg" alt="My Logo"></a>
+                        <nav class="contact"><img src="image/Icon - Menu.svg" alt="Mobile Menu"></nav>
+                </header>
+            <article class="img_background">
+                    <h1 class="header" >I’m Mariane<br> 
+                        Glad to see<br> you!</h1>
```

Comment:
[OPTIONAL] For best practices and semantic reasons, you do not need to wrap your header with a section tag. Kindly remove the section tag wrapping the header tag :pray:

Reply:
Noted and thank you